provisioning/device/src/interfaces.ts

```diff
@@ -171,8 +171,9 @@ export interface TpmRegistrationResult extends RegistrationResult {
  * @private
  */
 export interface TpmProvisioningTransport extends PollingTransport {
-  setSasToken(sasToken: string): void;
-  getAuthenticationChallenge(registrationInfo: TpmRegistrationInfo, callback: (err: Error, tpmChallenge?: TpmChallenge) => void): void;
+  setTpmInformation(ek: Buffer, srk: Buffer): void;
+  getAuthenticationChallenge(request: RegistrationRequest, callback: (err: Error, tpmChallenge?: TpmChallenge) => void): void;
+  respondToAuthenticationChallenge(request: RegistrationRequest, sasToken: string, callback: (err?: Error) => void): void;
```

Comment:
ok with this for now but there's another option i'd like to explore which is have a 'respond' method directly on the tpmChallenge - it'd be a pretty common pattern (think: http request/response) but i don't know if it'd help with the code.